### ISSUE TITLE
EUCLIDEAN_LOSS layer does not work

### ISSUE THREAD

**Issue by wangdelp** [BVLC/caffe#1666]
Hi, there

I changed the Loss function in the file to EUCLIDEAN_LOSS https://github.com/BVLC/caffe/blob/master/examples/mnist/lenet_train_test.prototxt

but the EUCLIDEAN_LOSS can not be identified when constructing the network. How to fix that bug? Thank you.

**Comment by AIML** [comment]
You may should change the learning rate(reduce it) since the value of  Euclidean_loss and softmax_loss is very different.

**Comment by shelhamer** [comment]
The Euclidean loss certainly does work and can be included in models. Double-check your prototxt definition and ask modeling questions on the [caffe-users](https://groups.google.com/forum/#!forum/caffe-users) mailing list. Thanks!
